app/forms/hyrax/generic_form.rb

```diff
@@ -11,7 +11,7 @@ class GenericForm < Hyrax::Forms::WorkForm
     self.required_fields = %i[title dcmi_type rights_statement]
 
     def primary_terms
-      required_fields + (OregonDigital::GenericMetadata::PROPERTIES.map(&:to_sym) - [:dcmi_type])
+      required_fields + (OregonDigital::GenericMetadata::PROPERTIES.map(&:to_sym) - required_fields)
```

Comment:
You could just remove the required_fields here right? instead of adding and then removing them

Reply:
No this makes sure that the required fields end up at the top. Otherwise they would be in the middle somewhere. For ease of testing.

Reply:
Can you be sure to make a comment about that so it's understandable "why"?

Reply:
Oh, I get it now, I wasn't seeing all the parentheses earlier. Thanks.